### PR TITLE
Document running alternatives, #152

### PR DESCRIPTION
**Pull request by patriknw**
References #152

### CODE REVIEW

File: docs/src/main/paradox/running.md [hunk 1]
```diff
@@ -74,10 +74,31 @@ For this example, we configure as many `ShardedDaemonProcess` as tags and we def
...
-### Running in Cluster Singleton
+Scala
+:  @@snip [CassandraProjectionDocExample.scala](/examples/src/test/scala/docs/cassandra/CassandraProjectionDocExample.scala) { #running-with-actor }
+
+Java
+:  @@snip [CassandraProjectionDocExample.java](/examples/src/test/java/jdocs/cassandra/CassandraProjectionDocExample.java) { #running-with-actor }
+
+Be aware of that the projection and its offset storage is based on that only one instance of the unique `ProjectionId`
+is running at a time. If more than one instance with the same `ProjectionId` are running concurrently they will
+overwrite each others offset storage with undefined and unpredictable results.
```
Comment (renatocaval) [inline docs/src/main/paradox/running.md:90]: I'm finding this sentence confusing. 

Does it better express what you had in mind? 

```suggestion
Be aware of that the projection and its offset storage is based on the `ProjectionId`. If more than one instance with the same `ProjectionId` are running concurrently they will
overwrite each others offset storage with undefined and unpredictable results.
```

Comment (renatocaval) [reply docs/src/main/paradox/running.md:90]: Can we use the ProjectionId on the actor path to avoid a second one to start?

Comment (patriknw) [reply docs/src/main/paradox/running.md:90]: good idea, I'll change to that
(although it can still be started by another parent causing duplicates)

Comment (renatocaval) [reply docs/src/main/paradox/running.md]: yes, only sharding can redeem us from our sins

File: examples/src/test/java/jdocs/cassandra/CassandraProjectionDocExample.java [hunk 2]
```diff
@@ -215,28 +222,78 @@ public static void illustrateAtLeastOnceFlow() {
...
-     public IllustrateRunningWithShardedDaemon() {
+    public IllustrateRunningWithShardedDaemon() {
+      //#running-with-daemon-process
+      ShardedDaemonProcess.get(system).init(
+          ProjectionBehavior.Command.class,
+          "shopping-carts",
+          ShoppingCart.tags.size(),
+          id -> ProjectionBehavior.create(projection(ShoppingCart.tags.get(id))),
+          ShardedDaemonProcessSettings.create(system),
+          Optional.of(ProjectionBehavior.stopMessage())
```
Comment (renatocaval) [inline examples/src/test/java/jdocs/cassandra/CassandraProjectionDocExample.java:242]:
```suggestion
          ProjectionBehavior.stopMessage()
```

We have Akka 2.6.6!

Comment (renatocaval) [reply examples/src/test/java/jdocs/cassandra/CassandraProjectionDocExample.java:242]: I will PR this one, let's merge it.

**Pull request merged**